packages/eslint-plugin-turbo/package.json

```diff
@@ -24,7 +24,6 @@
     "dist/**"
   ],
   "scripts": {
-    "release": "pnpm build && pnpm publish",
```

Comment:
Removing vestigial from prior iterations of our publishing flows. This isn't being used anymore.